backend/api/conferences/types.py

```diff
@@ -323,11 +323,12 @@ def ranking(self, info) -> Optional[RankRequest]:
             ):
                 return None
 
+            submissions = rank_request.rank_submissions.filter(
+                submission__topic__id=int(topic)
```

Comment:
```suggestion
                submission__topic__id=topic
```

I don't think we need to cast here, django should handle it 👍